### DISCUSSION TITLE
Change font size of one field

### DISCUSSION THREAD

**Discussion by thusharagokulnath** [py-pdf/pypdf#2725] [Q&A]
Any way to change the font size of one field in PDF generated using pypdf writer class

TIA

**Comment by pubpub-zz** [comment]
This feature is in the main  branch (#2636) but not yet part of release (we are facing some issues to finish the workflow)
You should be able to try it (see https://pypdf.readthedocs.io/en/stable/dev/testing.html#evaluate-a-pr-in-progress-version)
